spec/lib/danger/scm_source/git_repo_spec.rb

```diff
@@ -4,12 +4,10 @@
   describe "#exec" do
     it "run command with our env set" do
       git_repo = described_class.new
-      allow(git_repo).to receive(:default_env) { Hash("LANG" => "zh_TW.UTF-8") }
-      command = Gem.win_platform? ? "status && set LANG" : "status && echo $LANG"
+      allow(git_repo).to receive(:default_env) { Hash("GIT_EXEC_PATH" => "git-command-path") }
```

Comment:
[`GIT_EXEC_PATH`](https://git-scm.com/book/en/v2/Git-Internals-Environment-Variables) is useful in this case.

`git --exec-path` can check the value.